app/_how-tos/synchronize-with-git.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Synchronize Insomnia content with Git
+
+products:
+- insomnia
+
+tags:
+- documents
+- collections
+- mock-servers
+- environments
+- git
+
+tier: team
+
+prereqs:
+    inline:
+    - title: Git repository
+      content: |
+        Create an empty remote Git repository.
+      icon_url: /assets/icons/git.svg
+    - title: Insomnia workspace
+      content: |
+        Create a workspace in Insomnia. A workspace can be a [design document](/how-to/create-a-design-document), a [collection](), a [mock server](), or an [environment]().
+      icon_url: /assets/icons/git.svg
+
+tldr:
+    q: How can I push content from Insomnia to a Git repository?
+    a: Create an empty remote Git repository and an Insomnia workspace. In the workspace, click the button at the bottom of the left pane and connect the repository, then commit and push your content.
+---
```

Comment:
I'm trying to do this now and getting a "Loading of Git Repository Failed: Could not find HEAD." error. Did you get this error? I'm going to see if I can figure out how to fix it and then I can add a troubleshooting FAQ here about it

Reply:
No I didn't have any issues, I tried with GitHub, GitLab, and BitBucket, and it all worked as expected